templates/main.html

```diff
@@ -21,6 +21,17 @@
       </a>
       <h1 data-i18n="page-title">Google Code-in 2017 Current Leaders</h1>
     </div>
+    <div class="ribbon">
```

Comment:
Add the copyright notice `Copyright (c) 2016 Tim Holman` just above the code. So, that we don't infringe the rights.